app/models/data_request_log.rb

```diff
@@ -0,0 +1,24 @@
+class DataRequestLog < ApplicationRecord
+  belongs_to :data_request
+  belongs_to :user
+
+  validates :num_pages, numericality: { only_integer: true, greater_than_or_equal_to: 0 }
```

Comment:
I think receiving zero pages is a valid outcome for a data request and should be able to be accepted